service-mesh/control-plane/base/resource-templates/oauth-configmap.yaml

```
@@ -10,11 +10,11 @@ data:
       name: token
       generic_secret:
         secret:
-          inline_string: "${CLIENT_SECRET}"
+          inline_string: "$CLIENTS_SECRET" 
   hmac-secret.yaml: |-
     resources:
     - "@type": "type.googleapis.com/envoy.extensions.transport_sockets.tls.v3.Secret"
       name: hmac
       generic_secret:
         secret:
-          inline_bytes: "${CLIENT_HMAC}"
\ No newline at end of file
+          inline_bytes: "$HMACS_SECRET"
```

It's HMAC (Hash-based Message Authentication Code)

```suggestion
          inline_bytes: "$HMAC_SECRET"
```